docs/tutorials/api/datasets.ipynb

```diff
@@ -0,0 +1,1163 @@
+{
```

Comment:
<p>I'd say this is one API level  too "low". In the datasets tutorial we should only cover the attributes and methods of the <code>Dataset</code>  objects. The explanation of the <code>PSFMap</code>  does not belong here. Also the <code>PSFKernelMap</code>  object does no exist.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/gammapy/gammapy/pull/3488/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='18'/>